scripts/enforceVersions.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+if [[ $(pwd) =~ .+simorgh ]]; then
+  desired_node_version=$(cat .nvmrc)
+
+  if [[ $(node -v) != "$desired_node_version"* ]]; then
+    tput setaf 1
+    echo "Oh no! You're not using our version of node, please move to $desired_node_version"
+    tput sgr0
+    echo "This can be done by running 'nvm install $desired_node_version && nvm use $desired_node_version'"
```

Comment:
```suggestion
    echo "This can be done by running 'nvm install $desired_node_version && nvm use $desired_node_version'\n "
    tput sgr0
```

New line and set terminal back to white